packages/terra-status-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Replaced style `overflow: auto;` from `innerView` to `outerView`.
```

Comment:
This should describe what was broken/fixed. 

Something like:

> Fix glyph scrollbar appearing when there is enough room to display

Reply:
Updated here https://github.com/cerner/terra-core/pull/3392/commits/b960ae6e215b94422d0db6eb5ec69289524df161.